rules/okta_rules/okta_threatinsight_security_threat_detected.py

```diff
@@ -0,0 +1,13 @@
+from panther_base_helpers import okta_alert_context
+
+
+def rule(event):
+    return event.get("eventtype") == "security.threat.detected"
+
+
+def title(event):
+    return f"Okta Security [{event.get('severity', '')}] Threat Detected "
```

Comment:
```suggestion
    return f"Okta ThreatInsight identified potentially malicious behavior for {event.get('actor).get('alternateId')} "
```

Reply:
Added displayName instead of alternateId because IP Addresses are often the actor for threats and don't contain an alternateId